src/views/bill/_form.php

```diff
@@ -135,6 +136,8 @@
                                 <?= $form->field($model, "[$i]type_id")->widget(BillTypeVueTreeSelect::class, [
                                     'billTypes' => $billTypesList,
                                     'replaceAttribute' => 'type_id',
+                                    'deprecatedTypes' => Yii::$app->params['module.finance.bill.types']['deprecated.types'],
```

Comment:
```suggestion
                                    'deprecatedTypes' => Yii::$app->params['module.finance.bill.types']['deprecated.types'],
```

Add this property to `config.php` of this repo, because it should always exist.